src/main/java/thomasmccue/dbclientapp/dao/FirstLevelDivisionDao.java

```diff
@@ -0,0 +1,99 @@
+package thomasmccue.dbclientapp.dao;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import thomasmccue.dbclientapp.helper.JDBC;
+import thomasmccue.dbclientapp.model.FirstLevelDivision;
+
+import java.sql.*;
+
+public class FirstLevelDivisionDao {
+      public static ObservableList<String> getDiv1List() {
+          String sql = "SELECT CONCAT_WS(', ', Division_ID, Division) " +
+                  "FROM client_schedule.first_level_divisions WHERE Country_ID = ?;";
+          ObservableList<String> usList = FXCollections.observableArrayList();
+
+        try (Connection connection = JDBC.getConnection();
+             PreparedStatement preparedStatement = connection.prepareStatement(sql)) {
+
+                preparedStatement.setInt(1, 1);
+
+                try (ResultSet resultSet = preparedStatement.executeQuery()) {
+                    while (resultSet.next()) {
+                        usList.add(resultSet.getString(1));
+                    }
+                }
+            } catch (SQLException e) {
+                e.printStackTrace();
+                throw new RuntimeException(e);
+            }
+        return usList;
+        }
+
+    public static ObservableList<String> getDiv2List() {
+        String sql = "SELECT CONCAT_WS(', ', Division_ID, Division) " +
+                "FROM client_schedule.first_level_divisions WHERE Country_ID = ?;";
+        ObservableList<String> ukList = FXCollections.observableArrayList();
+
+        try (Connection connection = JDBC.getConnection();
+             PreparedStatement preparedStatement = connection.prepareStatement(sql)) {
+
+            preparedStatement.setInt(1, 2);
+
+            try (ResultSet resultSet = preparedStatement.executeQuery()) {
+                while (resultSet.next()) {
+                    ukList.add(resultSet.getString(1));
+                }
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+            throw new RuntimeException(e);
+        }
+        return ukList;
+    }
+
+    public static ObservableList<String> getDiv3List() {
+        String sql = "SELECT CONCAT_WS(', ', Division_ID, Division) " +
+                "FROM client_schedule.first_level_divisions WHERE Country_ID = ?;";
+        ObservableList<String> canList = FXCollections.observableArrayList();
+
+        try (Connection connection = JDBC.getConnection();
+             PreparedStatement preparedStatement = connection.prepareStatement(sql)) {
+
+            preparedStatement.setInt(1, 3);
+
+            try (ResultSet resultSet = preparedStatement.executeQuery()) {
+                while (resultSet.next()) {
```

Comment:
Lots of duplication here, you could have another method that takes the country id and then you could centralize the logic and not need to be repeating yourself here for each country.

Reply:
Fixed in my most recent commit to the branch